charts/backstage/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.15.1
+version: 0.16.0
```

Comment:
since this is a bugfix could you set `0.15.2` as version? 😄

Reply:
I am happy to do that, but in reality, it is not backward compatible.